vendor/github.com/hpe-storage/common-host-libs/storageprovider/csp/container_storage_provider.go

```diff
@@ -462,6 +462,36 @@ func (provider *ContainerStorageProvider) ExpandVolume(id string, requestBytes i
 	return response, err
 }
 
+// CreateVolume creates a volume on the CSP
+func (provider *ContainerStorageProvider) EditVolume(id string, opts map[string]interface{}) (*model.Volume, error) {
```

Comment:
Comment doesn't match the function name.

Reply:
This was fixed in latest common-host-libs. will update the package.